metaworld/envs/mujoco/multitask_env.py

```diff
@@ -158,6 +160,8 @@ def _augment_observation(self, obs):
         return obs
 
     def reset(self, **kwargs):
+        self.active_env.set_task(is_partially_observable=False)
```

Comment:
how do you make it partially-observable for ML-x?

Reply:
I didn't realize we wanted it to be partially observable. Some environments can't be solved with scripted policies under that condition. I suppose that may change now that Avnish and KR are changing goal sampling, but it's not exactly clear to me how all of that is working at the moment.

Reply:
ML-x will be partially-observable while MT-x will be fully-observable.

It's a given that the partially-observable versions will not be solvable by the scripted policies. It's fine, as long as the fully-observable versions are.